week1/chileshe.js

```diff
@@ -0,0 +1,64 @@
+"use strict";
+
+const http = require("http");
+const server = http.createServer();
+
+server.on("connection", () => {
+    console.log("Connected")
+});
+
+
+const PORT = 8080;
+
+server.listen(PORT, () => {
+    console.log("Listening on port", PORT);
+});
+
+
+let state = 10;
+
+server.on("request", (req, res) => {
+
+    res.setHeader("content-type", "text/html");
+
+
+    function writeState() {
```

Comment:
You're re-declaring these functions every time a new request comes in, which is unnecessary - the function definitions are the same every time.